### ISSUE TITLE
PUT /valuedescriptor/ can't change description

### ISSUE THREAD

**Issue by EvgenyOrekhov** [edgexfoundry/edgex-go#714]
Giving the following value descriptor:

```json
{
    "id": "5bc0aa89ee80aed44586cba7",
    "created": 1539353225739,
    "description": "device notification message",
    "modified": 1540211241919,
    "origin": 0,
    "name": "message",
    "min": null,
    "max": null,
    "defaultValue": null,
    "type": "S",
    "uomLabel": null,
    "formatting": "%s",
    "labels": null
}
```

After executing `PUT /valuedescriptor/`

```json
{
    "id": "5bc0aa89ee80aed44586cba7",
    "name": "message",
    "description": "new description",
    "type": "S",
    "uomLabel": null,
    "defaultValue": null,
    "min": null,
    "max": null,
    "formatting": "%s",
    "labels": []
}
```

I get the result: `true`

But the description doesn't change, after executing `GET /valuedescriptor/5bc0aa89ee80aed44586cba7` I get this:

```json
{
    "id": "5bc0aa89ee80aed44586cba7",
    "created": 1539353225739,
    "description": "device notification message",
    "modified": 1540211611294,
    "origin": 0,
    "name": "message",
    "min": null,
    "max": null,
    "defaultValue": null,
    "type": "S",
    "uomLabel": null,
    "formatting": "%s",
    "labels": null
}
```

**Comment by jduranf** [comment]
Thank you @EvgenyOrekhov, pull request #719 have been created in order to fix the issue.

**Comment by jduranf** [comment]
Fixed via #719
